apisix/plugins/proxy-mirror.lua

```diff
@@ -33,6 +33,12 @@ local schema = {
             type = "string",
             pattern = [[^/[^?&]+$]],
         },
+        mode = {
```

Comment:
As it isn't "mode" of the proxy-mirror, I think calling it "path_concat_mode" would be better.

Reply:
@spacewander Done. PTAL.